.docksal/src_files/Acquia_settings/.acquia/cloudapi.conf

```diff
@@ -0,0 +1 @@
+{"email":"mdyoung3@stanford.edu","key":"gOoBtv57btCoLt0EXa8yzmas\/ESORvy2dV1LVQ10W+V7TdCyVQuUR28GgvMeQV9bH5zFOuKqsjP4"}
```

Comment:
Probably don't want to commit an API key to a public repo. And you now probably should revoke that and re-generate.